src/rainflow.py

```diff
@@ -146,7 +146,8 @@ def count_cycles(series, ndigits=None, nbins=None, binsize=None, left=False, rig
     
     Returns
     -------
-    A sorted list containing pairs of cycle magnitude and count.
+    A sorted list containing pairs of cycle bin value and count.
+    The bin values represent the right-hand edge value of the bins.
     One-half cycles are counted as 0.5, so the returned counts may not be
```

Comment:
Please reword the docstring to:

A sorted list containing pairs of cycle magnitude and count.
One-half cycles are counted as 0.5, so the returned counts may not be
whole numbers. If binning is used the cycle magnitude corresponds
to the right edge of the bin.